functional/vfio/run.sh

```diff
@@ -109,6 +109,7 @@ EOF
 
 setup_configuration_file() {
 	local qemu_config_file="configuration-qemu.toml"
+	local qemu_virtiofs_config_file="configuration-qemu-virtiofs.toml"
```

Comment:
I can't find an `.in` file for `configuration-qemu-virtiofs.toml` in https://github.com/kata-containers/kata-containers/tree/2.0-dev/src/runtime/cli/config

this may not work